diff -dur -x '*~' resin-3.0.13.orig/configure resin-3.0.13/configure
--- resin-3.0.13.orig/configure	2005-03-21 23:58:43.000000000 +0100
+++ resin-3.0.13/configure	2005-06-05 18:08:55.000000000 +0200
@@ -8288,33 +8288,17 @@
    { (exit 1); exit 1; }; }
 fi
 
-#
-# Heuristics for finding the Apache configuration directory
-#
+# in PLD /etc/httpd and httpd.conf are unreadable for lusers,
+# and we should assume they're in /etc/httpd/httpd.conf 
 
 if test -z "${apache_conf}" -a -n "${apxs}"; then
   apache_confdir=`${apxs} -q SYSCONFDIR`
 
-  if test -r "${apache_confdir}"/httpd.conf; then
-    apache_conf=${apache_confdir}/httpd.conf
-  elif test -r "${apache_confdir}"/apache.conf; then
-    apache_conf=${apache_confdir}/apache.conf
-  elif test -r "${apache_confdir}"/httpsd.conf; then
-    apache_conf=${apache_confdir}/httpsd.conf
-  fi
-fi
-
-if test -z "${apache_conf}" -a -r "${apache_dir}/conf/httpd.conf"; then
-  apache_conf=${apache_dir}/conf/httpd.conf
-fi
-
-if test -z "${apache_conf}" -a -r "${apache_dir}/etc/httpd.conf"; then
-  apache_conf=${apache_dir}/etc/httpd.conf
 fi
 
-if test -n "$apache" -a ! -r "${apache_conf}"; then
+if test -n "$apache" -a ! -z "${apache_conf}"; then
   { { echo "$as_me:$LINENO: error: Can't find Apache module configuration ${apache_conf}" >&5
-echo "$as_me: error: Can't find Apache module configuration ${apache_conf}" >&2;}
+echo "$as_me: error: Apache module configuration not found/specified" >&2;}
    { (exit 1); exit 1; }; }
 fi
 #
